<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.PropertyExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-05-06T14:34:17" hostname="testing-docker-61402520-d4dd-4f1f-ac93-0a4d284e0ebf" time="0.005">
  <properties/>
  <testcase name="Exception should have proper formatted message[0]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.004"/>
  <testcase name="Exception should have proper formatted message[1]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message[2]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
